doc/common_pitfalls.rst

```diff
@@ -416,7 +416,7 @@ it will allow the estimator RNG to vary for each fold.
 **Cloning**
 
 Another subtle side effect of passing `RandomState` instances is how
-:func:`~sklearn.clone` will work::
+:func:`~sklearn.base.clone` will work::
 
     >>> from sklearn import clone
```

Comment:
I am fine importing from `sklearn.base` in the text above but we should also update this docstest snippet to be consistent.